src/libs/actions/IOU.ts

```diff
@@ -5808,6 +5817,10 @@ function setMoneyRequestParticipantsFromReport(transactionID: string, report: On
             ? [{accountID: 0, reportID: chatReport?.reportID, isPolicyExpenseChat: ReportUtils.isPolicyExpenseChat(chatReport), selected: true}]
             : (chatReport?.participantAccountIDs ?? []).filter((accountID) => currentUserAccountID !== accountID).map((accountID) => ({accountID, selected: true}));
 
+    if (iouType === CONST.IOU.TYPE.SPLIT) {
```

Comment:
+1, we're also removing the option to select participants from the split.

Reply:
I moved this logic to confirmation page since we're now allow skip confirmation step for QAB. The new design include the current user in split amounts section, that is the reason I add the current user to participants of transaction https://github.com/Expensify/App/issues/40379#issue-2249250830.